explorer/src/providers/cache.tsx

```diff
@@ -100,6 +101,12 @@ export function reducer<T, U>(
           ...entry,
           status: action.status,
           data: reconciler(entry?.data, action.data),
+          attempts:
```

Comment:
Confirmation attempts feels too specific to transactions to be added here. In the future, something like a retry counter for failed fetches would be nice here though

Reply:
Attempts in this case is just the number of successful fetches completed, not so much the number of confirmation attempts. Maybe attempts is the wrong name. Should this go elsewhere?